chart/openfaas/templates/gateway-dep.yaml

```diff
@@ -33,6 +33,9 @@ spec:
       {{- end }}
       containers:
       - name: gateway
+        resources:
+          requests:
+            memory: "120Mi"
```

Comment:
I would add `50m` CPU for the gateway and operator/faas-netes so HPA will work for this pod.

Reply:
Added